### PR TITLE
access to new control through web on add-on installation

### PR DESCRIPTION
**Pull request by prakharjoshi**
adding readme for the add-on

trying to pin the z3cform to 1.1.0

pinning few packages to remove the travis error

correcting the tests and adding functionality to access control panel through web

pin up plone.keyring

adding contenttypes layer for testing enviornment

correcting readme and removing unwanted tests

proper buildout

### CODE REVIEW

**Comment by prakharjoshi** [comment]
@gforcada  Sorry for the delay. I have missed it earlier. But now stashed the commits in a new branch. I guess we can merge that now.

File: src/experimental/safe_html_transform/tests/test_filter.py [hunk 1]
```diff
@@ -24,36 +24,41 @@ def setUp(self):
...
+        self.browser.open(self.portal_url + '/@@filter-controlpanel')
         self.browser.getControl(name="form.buttons.cancel").click()
         self.assertTrue(
             'Changes canceled.' in self.browser.contents)
 
+    def test_login(self):
+        self.browser.open(self.portal.absolute_url() + "/login_form")
+        self.browser.getControl(name='__ac_name').value = 'admin'
+        self.browser.getControl(name='__ac_password').value = 'admin'
+        self.browser.getControl(name='submit').click()
```
Comment (gforcada) [inline src/experimental/safe_html_transform/tests/test_filter.py]: @prakharjoshi what's this test for? I would remove it.

**Comment by gforcada** [comment]
Hi @prakharjoshi aside from that test that does not make sense I would say is ready to merge. Nice work!

**Comment by prakharjoshi** [comment]
@gforcada  Yeah that test was not relevant here but it contains the `READMe` for the repo and also some pinning of products as you can see in the squash commits above.

File: src/experimental/safe_html_transform/tests/test_filter.py (hunk 1 above)
Comment (prakharjoshi) [inline src/experimental/safe_html_transform/tests/test_filter.py]: @gforcada  Sorry for the extra test case. Yeah we can remove that right now.

**Comment by prakharjoshi** [comment]
@gforcada  Updated :D

**Pull request merged**